cdap-kubernetes/src/main/java/io/cdap/cdap/k8s/util/WorkloadIdentityUtil.java

```diff
@@ -96,7 +96,7 @@ public static void findOrCreateWorkloadIdentityConfigMap(CoreV1Api coreV1Api, St
     throws ApiException, IOException {
     // Check if workload identity config map already exists
     try {
-      coreV1Api.readNamespacedConfigMap(k8sNamespace, WORKLOAD_IDENTITY_CONFIGMAP_NAME, null, false, false);
```

Comment:
what is the ramification of ignoring these two false params?

Reply:
These are the query param values used when reading NameSpace config map, we had set these two params to false previously and is now removed. Here's the old documentation: 

- "exact" - Should the export be exact. Exact export maintains cluster-specific fields like 'Namespace'. Deprecated. Planned for removal in 1.18. (optional)
- "export" - Should this value be exported. Export strips fields that a user can not specify. Deprecated. Planned for removal in 1.18. (optional)

Docs: https://javadoc.io/static/io.kubernetes/client-java-api/7.0.0/io/kubernetes/client/openapi/apis/CoreV1Api.html#readNamespacedConfigMapWithHttpInfo-java.lang.String-java.lang.String-java.lang.String-java.lang.Boolean-java.lang.Boolean-